tests/baselines/reference/multiLinePropertyAccessAndArrowFunctionIndent1.types

```diff
@@ -6,12 +6,12 @@ return this.edit(role)
 >this.edit : any
 >this : typeof globalThis
 >edit : any
->role : any
+>role : error

```

Comment:
This is a weird change... this is the `role` parameter in this test's confusing return statement. Not sure why it went from an implicit any to an `error`.